.travis.yml

```diff
@@ -7,7 +7,7 @@ python:
   - "3.4"
   - "3.5"
 sudo: false
-script: nosetests test --verbose
+script: nosetests test --verbose --processes=4 --process-timeout=540
```

Comment:
I have seen you have try how many cores would use in Travis (https://travis-ci.org/yan12125/travis-ci-test/builds/87175866#L87), 16 is probably too much. It seems fine to use 4.